### PR TITLE
Data batcher script

### PR DESCRIPTION
**Pull request by khgitting**

### CODE REVIEW

File: bin/caqh_id_by_client [hunk 1]
```diff
@@ -0,0 +1,187 @@
...
+    :rtype: list[list[object]]
+    """
+    with get_connection(staging_env, use_dict_cursor) as conn:
+        cur = conn.cursor()
+        cur.execute(query, query_args)
+        return cur.fetchall()
+
+
+def caqh_query(staging_env=False):
+    """
```
Comment (mike-singh) [inline bin/caqh_id_by_client]: This is just me being curious - are these ```"""``` multi-line strings? 🤔

File: bin/caqh_id_by_client [hunk 2]
```diff
@@ -0,0 +1,187 @@
...
+
+    :param int client_id: a credsimple client id. Used as a filter to build CAQH batches for a
+        specific client. If no argument is passed, the default behavior is to return a dataframe
+        for all clients
+    :param pd.Timestamp: The time of the incident. Any elements that have been updated AFTER
+        incident_end_timestamp are included; any elements that have NOT been updated since
+        incident_end_timestamp are considered un-fixed and are kept/returned
+    """
+    if client_id:
+        client_ix = df.client_id == client_id
```
Comment (mike-singh) [inline bin/caqh_id_by_client]: ```df = dateframe```? what is ```ix``` 🤔 ?

File: bin/caqh_id_by_client [hunk 3]
```diff
@@ -0,0 +1,187 @@
...
+import time
+
+import click
+import pandas as pd
+import psycopg2
+from psycopg2.extras import RealDictCursor
+
+
+CORE_STAGING_HOST = 'staging-rds.credsimple.local'
+CORE_PROD_HOST = 'pr95k1q298xbe.cz8ldsglonio.us-east-1.rds.amazonaws.com'
```
[the remark this answers is not in the document]
Comment (khgitting) [reply bin/caqh_id_by_client]: Ya, no problem being hardcoded. And yes, you must be on the vpn.

File: bin/caqh_id_by_client (hunk 1 above)
Comment (khgitting) [reply bin/caqh_id_by_client]: Yep

File: bin/caqh_id_by_client (hunk 2 above)
Comment (khgitting) [reply bin/caqh_id_by_client]: Ya, df is a pretty common variable name for a dataframe

ix is an alternative for idx. client_ix is a series that will be used to index into the dataframe

**Pull request merged**